yorkie/src/main/kotlin/dev/yorkie/util/SplayTreeSet.kt

```diff
@@ -0,0 +1,364 @@
+package dev.yorkie.util
+
+import com.google.common.annotations.VisibleForTesting
```

Comment:
This is a cool feature. There were cases where other SDKs exposed privates to the public because of test code.

Reply:
this annotation is very convenient, but doc states that it is not a good practice for public and protected visibility. :(
> Do not use this interface for public or protected declarations: it is a fig leaf for bad design, and it does not prevent anyone from using the declaration---and experience has shown that they will.

but since it is used on an internal class here, we are good to go :)